carmm/analyse/meshgrid/meshgrid_void.py

```diff
@@ -0,0 +1,134 @@
+def void_find(MeshObject, Atom, coarseness=1):
+    """
+    Defines the void centres and radii of points across the meshgrid. Coarseness factor reduces number of probe points
+    by skipping over points in the grid. Uses 99.99 as a junk value.
+    Args:
+        MeshObject: MeshgridObject object
+            Contains meshgrid and associated unit cell information.
+        Atom: Atoms object
```

Comment:
Atoms object is a list of Atom objects, misleading name

Reply:
Ditto. Resolved.